llvm/lib/Target/EraVM/MCTargetDesc/EraVMMCCodeEmitter.cpp

```diff
@@ -60,15 +66,66 @@ class EraVMMCCodeEmitter : public MCCodeEmitter {
                         SmallVectorImpl<MCFixup> &Fixups,
                         const MCSubtargetInfo &STI) const;
 
+  uint64_t fixOpcode(const MCInst &MI, uint64_t EncodedInstr,
```

Comment:
Documentation is welcome here.

Reply:
Added an explanation and gave the function more descriptive name in 8b42a8d4ca7550cb137fc131fa52057037df1847.